dash-pipeline/Makefile

```diff
@@ -592,7 +592,7 @@ DOCKER_SAI_CHALLENGER_CLIENT_BLDR_IMG = $(DOCKER_SAI_CHALLENGER_CLIENT_BLDR_IMG_
 
 docker-saichallenger-client-bldr:
 	{ [ x$(ENABLE_DOCKER_PULL) == xy ] && docker pull $(DOCKER_SAI_CHALLENGER_CLIENT_BLDR_IMG); } || \
-		{ pushd $(SAI_CHALLENGER_PATH) && ./build.sh -i client && popd; \
+		{ pushd $(SAI_CHALLENGER_PATH) && git submodule update --init && ./build.sh -i client && popd; \
```

Comment:
Looks like CI should be fixed to initialize submodules recursively.. Once this is done, we should revert this change.

Reply:
> Looks like CI should be fixed to initialize submodules recursively.. Once this is done, we should revert this change.

Hi @andriy-kokhan, can you be more specific which CI workflows require the submodule init? If this is the case, why not modify those instead of making an interim change to Makefile?

Also, how did you arrive at this conclusion? I don't see any CI workflows triggered by this PR, not even in your fork. I would request that you enable CI actions in your fork and ensuring the CI workflows all work. In the future, this should be done before submitting a PR to find errors in the fork first, thanks.

Reply:
I meant this PR's checks that already passed.
Initially, the failure was on "DASH-docker-saichallenger-client-bldr-img / Build dash-saichallenger-client-bldr-image (pull_request) Successful in 18m" check because PTF submodule of SAI-Challenger was not initialized.

Reply:
I'm not familiar with DASH checks yet. I think here is the problem https://github.com/Azure/DASH/blob/f3f335e9d8f8aaa452e1971d335f87966d449cd1/.github/workflows/dash-saichallenger-client-bldr-docker.yml#L41.
Let me check.

Reply:
Fixed